mirrord/protocol/src/tcp.rs

```diff
@@ -73,6 +73,14 @@ pub enum DaemonTcp {
     SubscribeResult(RemoteResult<Port>),
     HttpRequest(HttpRequest<Vec<u8>>),
     HttpRequestFramed(HttpRequest<InternalHttpBody>),
+    HttpRequestChunked(Result(ChunkedRequest, hyper::Error)),
+}
+
+/// Contents of a chunked message from server
+pub enum ChunkedRequest {
+    Start(HeaderMap),
```

Comment:
This should probably be `HttpRequest<()>` - the same as in `DaemonTcp::HttpRequest` and `DaemonTcp::HttpRequestFramed`, just without the body. We need more info than just the headers, e.g. path, original destination port, request id, probably more